orchestration/main.go

```diff
@@ -83,10 +85,16 @@ func main() {
 			LocalFolder: paths.host.source,
 			MetaFolder:  paths.host.meta,
 			KeyFile:     paths.host.key,
+			Update: reuseScmCheckout,
 			Env:         env,
 		},
 	}
-	if err := f.Fetch(containerLogger); err != nil {
```

Comment:
Ok. I think we should at least have a warning log to tell why we do a fresh checkout